src/backends/inspectdr.jl

```diff
@@ -0,0 +1,411 @@
+    :markerstrokewidth, :markerstrokecolor, :markerstrokealpha,
+    :markerstrokestyle, #Causes warning not to have it... what is this?
+    :fillcolor, :fillalpha, #:fillrange,
+#    :bins, :bar_width, :bar_edges, :bar_position,
+    :title, :title_location, :titlefont,
+    :window_title,
+    :guide, :lims, :scale, #:ticks, :flip, :rotation,
+    :tickfont, :guidefont, :legendfont,
+    :grid, :legend, #:colorbar,
+#    :marker_z,
+#    :line_z,
+#    :levels,
+ #   :ribbon, :quiver, :arrow,
+#    :orientation,
+    :overwrite_figure,
+#    :polar,
+#    :normalize, :weights,
+#    :contours, :aspect_ratio,
+    :match_dimensions,
+#    :clims,
+#    :inset_subplots,
+    :dpi,
+#    :colorbar_title,
+  ])
+const _inspectdr_style = [:auto, :solid, :dash, :dot, :dashdot]
+const _inspectdr_seriestype = [
+        :path, :scatter #, :steppre, :steppost, :shape,
+    ]
+#see: _allMarkers, _shape_keys
+const _inspectdr_marker = Symbol[
+    :none,
+    :auto,
+    :circle,
+    :rect,
+  :diamond,
+#  :hexagon,
+  :cross,
+  :xcross,
+  :utriangle,
+  :dtriangle,
+  :rtriangle,
+  :ltriangle,
+#  :pentagon,
+#  :heptagon,
+#  :octagon,
+#  :star4,
+#    :star5,
+#  :star6,
+#  :star7,
+  :star8,
+#  :vline,
+#  :hline,
+  :+,
+  :x,
+]
+
+const _inspectdr_scale = [:identity, :ln, :log2, :log10] #Does not really support ln, (plot using log10 instead).
+
+#Do we avoid Map to avoid possible pre-comile issues?
+function _inspectdr_mapglyph(s::Symbol)
+    s == :rect && return :square
+    s == :utriangle && return :uarrow
+    s == :dtriangle && return :darrow
+    s == :ltriangle && return :larrow
+    s == :rtriangle && return :rarrow
+    s == :xcross && return :diagcross
+    s == :star8 && return :*
+
+#= Actually supported:
+    :square, :diamond,
+    :uarrow, :darrow, :larrow, :rarrow, #usually triangles
+    :cross, :+, :diagcross, :x,
+    :circle, :o, :star, :*,
+=#
+
+    return s
+end
+
+# py_marker(markers::AVec) = map(py_marker, markers)
+function _inspectdr_mapglyph(markers::AVec)
+    warn("Vectors of markers are currently unsupported in InspectDR.")
+    _inspectdr_mapglyph(markers[1])
+end
+
+_inspectdr_mapglyphsize(v::Real) = v
+function _inspectdr_mapglyphsize(v::Vector)
+    warn("Vectors of marker sizes are currently unsupported in InspectDR.")
+    _inspectdr_mapglyphsize(v[1])
+end
+
+_inspectdr_mapcolor(v::Colorant) = v
+function _inspectdr_mapcolor(g::PlotUtils.ColorGradient)
+    warn("Vectors of colors are currently unsupported in InspectDR.")
+    #Pick middle color:
+    _inspectdr_mapcolor(g.colors[div(1+end,2)])
+end
+
+#Hack: suggested point size does not seem adequate relative to plot size, for some reason.
+_inspectdr_mapptsize(v) = 1.5*v
+
+# ---------------------------------------------------------------------------
+#InspectDR-dependent structures and method signatures.
+#(To be evalutated only once ready to load module)
+const _inspectdr_depcode = quote
+
+import InspectDR
+export InspectDR
```

Comment:
This is not allowed.  It would require adding InspectDR as a dependency of Plots, which is not an option.  Search for `export` in other backends to see how it's handled.